iml-api/src/graphql.rs

```diff
@@ -141,6 +142,86 @@ impl Deref for SortDir {
     }
 }
 
+#[derive(GraphQLEnum, sqlx::Type)]
+#[repr(i16)]
+enum MessageClass {
+    Normal = 0,
+    Lustre = 1,
+    LustreError = 2,
+    Copytool = 3,
+    CopytoolError = 4,
+}
+
+impl From<i16> for MessageClass {
```

Comment:
I think using `TryFrom` would be better than panicking if we can't find a conversion.

Reply:
I was looking at that, it doesn't bearthe weight IMO. The place where conversion is done doesn't support errors and it's too much for conversion of `LogMessageRecord` to `LogMessage` considering this is one-way and DB consistency is guaranteed.